### ISSUE TITLE
Add option to specify a name for IndexedDB store

### ISSUE THREAD

**Issue by alignsoft** [cyrilletuzi/angular-async-local-storage#29]
There's a high probability of running multiple apps in the same browser using this storage library, and if they all share the same default name 'ngStorage', each app has the potential to be writing over data from other apps.

It would be extremely useful to be able to specify a store name, perhaps by using something like:

```
    LocalStorageModule.withConfig({
      name: 'ngAppOneStore'
    }),

```
In the app.module?

**Comment by cyrilletuzi** [comment]
The browser APIs for local storage (`localStorage` and `indexedDb`) are restricted to the same origin (meaning each subdomain has its own scope).

There is a problem only if there are multiple apps on the same subdomain, which is likely a scenario where you _do_ want to share the same information.

Are you in the case of multiple apps on the same subdomain, but wanting to separate data for each app?

**Comment by cyrilletuzi** [comment]
OK, I'll plan to add this option.

Do you plan to migrate quickly to Angular 6 once released, or would you need this for v5?

**Comment by alignsoft** [comment]
Fantastic - thanks! We've been keeping all our apps current with the 5.x releases as they've come out, but I don't know how quickly we'll be able to migrate them all to 6, so if this can be incorporated into the v5 version, that would be very much appreciated!

**Comment by cyrilletuzi** [comment]
@alignsoft Could you please test the new beta? I've added unit tests, but due to the nature of what's happening, I would be more confident with tests on a real case scenario.

Instructions:
1. `npm install @ngx-pwa/local-storage@beta`
2. In your `AppModule`, add:

```typescript
import { localStorageProviders } from '@ngx-pwa/local-storage';

@NgModule({
  providers: [
    localStorageProviders({ prefix: 'mycustomprefix' })
  ]
})
export class AppModule {}
```

Thanks.

**Comment by alignsoft** [comment]
I'm getting the following error: 
`"[ts] Module '".../node_modules/@ngx-pwa/local-storage/local-storage"' has no exported member 'localStorageProviders'." `
with `"@ngx-pwa/local-storage": "^5.3.0-beta.0"`, installed. 

This is in the app.module: `import { localStorageProviders } from '@ngx-pwa/local-storage';`

**Comment by cyrilletuzi** [comment]
@alignsoft I can't reproduce this error, I checked the published files and `localStorageProviders` is here with no typo, and I can import it in a project (there is another though, but not related).

Are you sure the 5.3.0-beta.0 is installed? Have you checked directly in `node_modules` files (not just in `package.json`)?

**Comment by cyrilletuzi** [comment]
New beta released with the unrelated issue fixed, just re-run:
`npm install @ngx-pwa/local-storage@beta`

**Comment by alignsoft** [comment]
It picks it up now with beta.1 - I'll test and let you know if I run into any issues. Cheers!

**Comment by alignsoft** [comment]
Seems to be working perfectly - no issues. Thanks for implementing this so quickly, and for all your work on this project - very much appreciated.

**Comment by cyrilletuzi** [comment]
Released in 5.3 and 6.0-rc.0.

Feel free to contact if you need an Angular training/course for you team. ;)

**Comment by cyrilletuzi** [comment]
Hi @alignsoft, configuration for this feature will change in v8, can you test it's still ok with v8 beta? Instructions in #76.
